.github/workflows/diff_report.yml

```diff
@@ -104,6 +104,17 @@ jobs:
         ref: master
         path: checkstyle
         fetch-depth: 0
+
+     # set main checkstyle repo as an upstream
+     # Diff report will be generated taking upstream's master branch as a base branch
+     - name: set upstream
+       run: |
+         cd checkstyle
+         bash
+         MAIN_REPO_GIT_URL="https://github.com/checkstyle/checkstyle.git"
+         git remote add upstream-master $MAIN_REPO_GIT_URL
```

Comment:
Make it
`git remote add upstream $MAIN_REPO_GIT_URL`

As you add whole repository, not a just one branch of it.

Reply:
@romani Done